index.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const cors = require('cors');
+const bcrypt = require('bcryptjs');
+
+const db = require('./dbAccess');
+
+const server = express();
+
+server.use(express.json());
+server.use(cors());
+
+server.post('/api/register', (req, res) => {
+    const creds = req.body;
+    const hash = bcrypt.hashSync(creds.password, 10);
+    creds.password = hash;
+
+    db('users')
+        .insert(creds)
+        .then(ids => {
+            const id = ids[0];
+            res.status(201).json(id);
+        })
+        .catch(err => res.status(500).send(err));
```

Comment:
I'd recommend console.logging the error in your `catch()` block. The error tends to be shortened in PostMan, and conceptually, it's not the client's job to fix your server, so they don't really need to know the exact error.

```js
// snippet
.catch(err => {
    console.log('/api/register POST error:', err);
    res.status(500).send('Please try again later.');
});
```